client/src/api/v1/index.ts

```diff
@@ -3,9 +3,15 @@ import axios, { AxiosInstance } from 'axios';
 import { Post } from 'models/post';
 import { User } from 'models/user';
 import { UserAttributes } from 'models/user';
+import { PostTag } from 'models/PostTags';
 
 export type PostUser = Post & {
+  likeCount: number;
+  doesUserLike: boolean;
   User: { id: string; firstName: string; lastName: string };
+  Tags: {
+    text: string & { PostTags: PostTag }; // sequelize pluarlizes name
```

Comment:
No, it was weird. I was doing manual testing and it returns a Tags object with text, and then a PostTags object. Maybe because I put `attributes: "text"` in my query? I'll look into the query to see if it can be cleaned up.

Reply:
We define attributes for the other queries and they are fine.